### ISSUE TITLE
[Help Wanted] Contributions to the Wiki

### ISSUE THREAD

**Issue by HugoBDesigner** [HugoBDesigner/Mari0-SE-Bugfixes#17]
I think it is a good idea to have wiki pages for different aspects of SE, given that we have a platform built specifically for that. My current proposals for wiki pages are:

- [ ] Custom enemies;
- [ ] Custom textures;
- [ ] Tilesets and tile properties;
- [ ] Animated tiles;
- [ ] In-game animations;
- [ ] Custom music;
- [ ] Logic items/entities;
- [ ] Custom characters;
- [ ] Custom backgrounds/foregrounds;
- [ ] Better Editor features;
- [ ] Mappack creation/packing.

Suggestions for wiki pages are welcome, be it new pages or the splitting of existing/suggested pages. Community contribution is very much appreciated for this particular task.

**Comment by masterminer176** [comment]
Would these pages be for describing the aspects of CE? Or would they be a repository for CE assets?

**Comment by HugoBDesigner** [comment]
They'd describe aspects of CE, such as tutorials. If users didn't know, for instance, how to make custom tilesets, the wiki would tell them everything: file name, file format, directory, tile sizes, tile properties, animated tiles, etc.

**Comment by Frostplays-ZERO** [comment]
The actual stabyourself wiki is pretty fully featured in this regard, right?

**Comment by HugoBDesigner** [comment]
I suppose so, but given this is an open project (and arguably more easily accessible), we can make it more comprehensive and thorough. Not to mention new features will require new knowledge too

**Comment by Frostplays-ZERO** [comment]
Yeah. I suppose that other issue about editor shortcuts (too lazy to tag it) is just one thing the STYS wiki doesn't cover. And having the wiki directly linked to the github would be cool too.

**Comment by masterminer176** [comment]
*bump*
Is there open application to help on this? There's no pages and I might be able to help with it

**Comment by Frostplays-ZERO** [comment]
May I volunteer as well? It might make this more accessible to new people.

**Comment by HugoBDesigner** [comment]
There are a couple ways you can contribute:
The first would be by posting an issue (or a comment in this issue) with the details of the page you want to create/edit.
The second would be by forking the Wiki, editing, then making a pull request (that might be a bit advanced for most users, though).
Alternatively, I could allow certain users to edit the Wiki, but that is not ideal in a public platform, even with rollbacks, so I'm reserving that for when other options are deemed impracticable.

I unfortunately haven't had much time to edit the Wiki myself, and I'm sorry the repo is kind of "abandoned", but as long as there are people willing to help, I'll try to facilitate it. With new contributions, I could add more members to the group and grant them edit permissions.

**Comment by magnusviri** [comment]
I've already forked the project but that didn't include the wiki. I don't see any way for me to fork the wiki actually. I've already started documenting stuff on my computer because I'm brand new to Mari0 and I basically don't know how to do anything so I've been writing it down for myself. There's this wiki: https://stabyourself.fandom.com/wiki/Mari0. Who controls it?

**Comment by magnusviri** [comment]
I don't see any way to edit it from the Mari0-CE page. I cloned it locally, made a change, tried to push it but was denied because I don't have permission. I created my own wiki but I see no way to push it to this repository's wiki. I can't see any way to contribute.

**Comment by HugoBDesigner** [comment]
@magnusviri Seems like, in order to push to the Wiki, one must be a member, which is quite a shame. I've invited you as a member, hopefully that'll sort it out. Sorry for the misunderstanding.
